server/src/routes/auth/signIn/signIn.js

```diff
@@ -0,0 +1,30 @@
+import bcrypt from 'bcryptjs';
+import User from '../../../models';
+import { signInValidation } from './validateSignIn';
+import { StatusCodes } from 'http-status-codes';
+import jwt from 'jsonwebtoken';
+
+
+/**
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+
+ const signIn = async (req, res) => {
+ const { error } = signInValidation(req.body)
+ if (error) return res.status(StatusCodes.BAD_REQUEST).send(error.details[0].message);
+
+ const user = await User.findOne({ email: req.body.email });
+ if (!user) return res.status(StatusCodes.BAD_REQUEST).send('Email is not found');
```

Comment:
We have to agree on how to send errors currently we have 3 different approaches which will make error handling on frontend way more difficult then it needs to be